src/Integration/MefServices/VsSessionHost.cs

```diff
@@ -297,25 +288,16 @@ private BindingConfiguration SafeGetBindingConfig()
 
         private void RegisterLocalServices()
         {
-            this.localServices.Add(typeof(ICredentialStoreService), new Lazy<ILocalService>(() => credentialStoreService));
-
-            this.localServices.Add(typeof(IConfigurationPersister), new Lazy<ILocalService>(GetConfigurationPersister));
```

Comment:
This is the only functional change in the method - deleting the service registration.